src/ray/gcs/redis_module/ray_redis_module.cc

```diff
@@ -505,6 +505,91 @@ int TableLookup_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int
   return REDISMODULE_OK;
 }
 
+// The deleting helper function.
+static Status DeleteKeyHelper(RedisModuleCtx *ctx, RedisModuleString *prefix_str,
+                              RedisModuleString *id_data) {
+  RedisModuleKey *delete_key = nullptr;
+  RAY_RETURN_NOT_OK(
+      OpenPrefixedKey(&delete_key, ctx, prefix_str, id_data, REDISMODULE_READ));
+  if (delete_key == nullptr) {
+    return Status::RedisError("Non-existing key");
+  }
+  auto key_type = RedisModule_KeyType(delete_key);
+  RedisModule_CloseKey(delete_key);
```

Comment:
Do we need to do this because we can't delete the key until it has been closed? Otherwise Redis automatic memory management should take care of it. If we need this line then please add a comment explaining why we need it.

Reply:
I removed this line.